specviz/widgets/plotting.py

```diff
@@ -17,18 +17,16 @@
 from ..core.items import PlotDataItem
 from ..core.models import PlotProxyModel
 
-from .linelists_window import LineListsWindow
-from ..core.linelist import ingest
-from ..core.linelist import LineList, WAVELENGTH_COLUMN, ID_COLUMN
-from .line_labels_plotter import LineLabelsPlotter
 
 __all__ = ['PlotWindow', 'PlotWidget']
 
 
 class PlotWindow(QMdiSubWindow):
     """
-    Displayed plotting subwindow available in the QMdiArea.
+    Displayed plotting subwindow available in the `QMdiArea`.
```

Comment:
Don't add backticks here - this is what's causing the error. Single backticks mean a proper API reference. Use double backticks if you want it to look like code.

Reply:
Done.